### PR TITLE
Refactor celery configuration

### PR DESCRIPTION
**Pull request by Uxio0**
@fmrsabino 😉

### CODE REVIEW

File: config/settings/base.py [hunk 1]
```diff
@@ -204,12 +204,13 @@
...
+
+CELERYD_CONCURRENCY = env("CELERYD_CONCURRENCY", default=500)
+CELERYD_POOL = env("CELERYD_POOL", default="gevent")
 # http://docs.celeryproject.org/en/latest/userguide/configuration.html#std:setting-broker_url
 CELERY_BROKER_URL = env("CELERY_BROKER_URL", default="django://")
 # https://docs.celeryproject.org/en/stable/userguide/optimizing.html#broker-connection-pools
-CELERY_BROKER_POOL_LIMIT = env(
-    "CELERY_BROKER_POOL_LIMIT", default=env("CELERYD_CONCURRENCY", default=500)
-)
+CELERY_BROKER_POOL_LIMIT = env("CELERY_BROKER_POOL_LIMIT", default=CELERYD_CONCURRENCY)
```
Comment (fmrsabino) [inline config/settings/base.py:213]: Should this be `BROKER_POOL_LIMIT`? https://docs.celeryproject.org/en/stable/userguide/configuration.html?highlight=CELERYD_CONCURRENCY#broker-pool-limit

File: config/settings/base.py [hunk 2]
```diff
@@ -204,12 +204,13 @@
 INSTALLED_APPS += [
     "django_celery_beat",
 ]
+
+CELERYD_CONCURRENCY = env("CELERYD_CONCURRENCY", default=500)
```
Comment (fmrsabino) [inline config/settings/base.py:208]: For this one, shouldn't it match the number of CPU cores available (default) on the host? Wondering how `500` would affect the number of worker processes spawn per host 🤔

File: config/settings/base.py (hunk 1 above)
Comment (Uxio0) [reply config/settings/base.py:213]: Celery on Python uses the `CELERY` namespace for every configuration variable

File: config/settings/base.py (hunk 2 above)
Comment (Uxio0) [reply config/settings/base.py:208]: But it was a good catch 😉

**Review by fmrsabino — APPROVED** [review]

**Pull request merged**